app/code/Magento/Inventory/Test/Acceptance/ActionGroup/FillMainProductFormInMultiSourceModeActionGroup.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This ActionGroup should live in `Magento_CatalogInventory` module

Reply:
@okolesnyk I am not  sure in it. Because we have similar `ActionGroup` in `Magento_CatalogInventory`, but this one doesn't fill `quantity` and `stock_status` as this field is absent in MSI mode. So this it is needed for MSI mode only. 
Do you still think that it should be moved to `Magento_CatalogInventory` ?

Reply:
@RomaKis  you not wrong, but not right either.
Talked to Valeriy I figured out that this action group represents functionality defined in `Magento_InventoryCatalogAdminUi` module. So please move this action group to that module.

Some reference where `qty` and `stock` fields removing `\Magento\InventoryCatalogAdminUi\Ui\DataProvider\Product\Form\Modifier\Quantity::modifyMeta`